org.bzdev.CVRDecode.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.bzdev.CVRDecode
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+command: cvrdecode.sh
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+   - org.freedesktop.Sdk.Extension.openjdk11
+modules:
+   - name: openjdk
+     buildsystem: simple
+     build-commands:
+        - /usr/lib/sdk/openjdk11/install.sh
+   - name: cvrdecode
+     buildsystem: simple
+     build-commands:
+       - install -D cvrdecode.sh /app/bin/cvrdecode.sh
+       - install -D cvrdecode.jar /app/cvrdecode.jar
+       - install -D libbzdev-base.jar /app/libbzdev-base.jar
+       - install -D libbzdev-desktop.jar /app/libbzdev-desktop.jar
+       - install -D core.jar /app/core.jar
+       - install -D javase.jar /app/javase.jar
+       - install -D org.bzdev.CVRDecode.desktop /app/share/applications/org.bzdev.CVRDecode.desktop
+       - install -D appicon256.png /app/share/icons/hicolor/256x256/apps/org.bzdev.CVRDecode.png
+       - install -D appicon128.png /app/share/icons/hicolor/128x128/apps/org.bzdev.CVRDecode.png
+       - install -D appicon64.png /app/share/icons/hicolor/64x64/apps/org.bzdev.CVRDecode.png
+       - install -D appicon48.png /app/share/icons/hicolor/48x48/apps/org.bzdev.CVRDecode.png
+       - install -D appicon32.png /app/share/icons/hicolor/32x32/apps/org.bzdev.CVRDecode.png
+       - install -D appicon16.png /app/share/icons/hicolor/16x16/apps/org.bzdev.CVRDecode.png
+       - install -D appdata.xml /app/share/metainfo/org.bzdev.CVRDecode.metainfo.xml
+       - install -D manpage /app/manpage
+     sources:
+        - type: archive
+          url: https://billzaumen.github.io/cvrdecode/flatpak.tar.gz
+          sha256: d61fc439c13700d8ea22b444b79c7790c15a1eab761eec625570e713ed06eea3
+finish-args:
+     - "--env=PATH=/app/jre/bin:/usr/bin:/app/bin"
+     - "--share=network"
+     - "--socket=wayland"
+     - "--socket=fallback-x11"
```

Comment:
sadly java doesn't really support wayland so you'll need to remove --socket=wayland and --socket=fallback-x11. It should work fine under xwayland (I tested on a wayland first distro)

Other than this, lgtm

Reply:
(you do that with `flatpak run --nosocket=wayland org.bzdev.CVRDecode` when running on a distro like the latest fedora btw in case you're looking to test yourself)